templates/resources/details.md

```diff
@@ -3,17 +3,13 @@ title: {Document title}
 type: Details
```

Comment:
The document could be removed or archived

Reply:
Removed :)